packages/suite/src/hooks/wallet/useUnstakeEthForm.ts

```diff
@@ -0,0 +1,230 @@
+import { createContext, useCallback, useContext, useEffect, useMemo } from 'react';
+import { useForm, useWatch } from 'react-hook-form';
+
+import { fromFiatCurrency, getFeeLevels, toFiatCurrency } from '@suite-common/wallet-utils';
+
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import {
+    CRYPTO_INPUT,
+    FIAT_INPUT,
+    OUTPUT_AMOUNT,
+    UseStakeFormsProps,
+} from 'src/types/wallet/stakeForms';
+import { mapTestnetSymbol } from 'src/utils/wallet/coinmarket/coinmarketUtils';
+import { AmountLimits } from 'src/types/wallet/coinmarketCommonTypes';
+
+import { useStakeCompose } from './form/useStakeCompose';
+import { selectLocalCurrency } from 'src/reducers/wallet/settingsReducer';
+import {
+    CONTRACT_POOL_ADDRESS,
+    MIN_ETH_AMOUNT_FOR_STAKING,
+    STAKED_ETH_WITH_REWARDS,
+} from 'src/constants/suite/ethStaking';
+
+import { signTransaction } from 'src/actions/wallet/stakeActions';
+import {
+    PrecomposedTransactionFinal,
+    UnstakeContextValues,
+    UnstakeFormState,
+} from '@suite-common/wallet-types';
+import { getStakeFormsDefaultValues } from 'src/utils/suite/stake';
+import { useFormDraft } from './useFormDraft';
+import useDebounce from 'react-use/lib/useDebounce';
+import { isChanged } from '@suite-common/suite-utils';
+
+const defaultValues = getStakeFormsDefaultValues({
+    address: CONTRACT_POOL_ADDRESS,
+    ethereumStakeType: 'unstake',
+}) as UnstakeFormState;
+
+export const UnstakeEthFormContext = createContext<UnstakeContextValues | null>(null);
+UnstakeEthFormContext.displayName = 'UnstakeEthFormContext';
+
+export const useUnstakeEthForm = ({
+    selectedAccount,
+}: UseStakeFormsProps): UnstakeContextValues => {
+    const dispatch = useDispatch();
+
+    const fiat = useSelector(state => state.wallet.fiat);
+    const localCurrency = useSelector(selectLocalCurrency);
+    const fees = useSelector(state => state.wallet.fees);
+
+    const { account, network } = selectedAccount;
+    const { symbol } = account;
+
+    const symbolForFiat = mapTestnetSymbol(symbol);
+    const fiatRates = fiat.coins.find(item => item.symbol === symbolForFiat);
```

Comment:
If you rebase to current develop, please change this to 

```
    const currentRate = useSelector(state =>
        selectFiatRatesByFiatRateKey(state, getFiatRateKey(symbol, localCurrency), 'current'),
    );
```

if possible ofc

Reply:
NP. I can do it in the next PR or right here. If you would like the change in this PR, let's do it in the last turn in order not to do rebase in the middle of the review.